### PR TITLE
Fix wrong tutorial location

### LINKED ISSUE #194: Template file not found: notestutorial-main
I've been trying to learn to develop a Nextcloud app. I tried to go through with the steps but on my Nextcloud 19 instance following [this tutorial](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html) always results in a broken app right after the database migration. Then I tried to download the finished app from this repo, but when I import the app I get the following error:
image
I suspect this is because the nextcloud app generator only supporting apps up to version 15 and old apps not working on version 19. Am I doing something wrong?

Nextcloud version: 19.0.1

### PR DESCRIPTION
**Pull request by skjnldsv**
fix #194

### CODE REVIEW

File: lib/Db/NoteMapper.php [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 namespace OCA\NotesTutorial\Db;
 
+use OCA\NotesTutorial\AppInfo\Application;
```
Comment (ChristophWurst) [inline lib/Db/NoteMapper.php:5]: is it used, though? I think you can remove it

**Review by ChristophWurst — CHANGES_REQUESTED** [review]
The rest looks fine :)

File: lib/Db/NoteMapper.php (hunk 1 above)
Comment (skjnldsv) [reply lib/Db/NoteMapper.php:5]: Oupsy!

Comment (skjnldsv) [reply lib/Db/NoteMapper.php:5]: Done

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**